stdlib/builtins.pyi

```diff
@@ -1329,7 +1336,7 @@ def round(number: SupportsRound[Any]) -> int: ...
 def round(number: SupportsRound[Any], ndigits: None) -> int: ...
 @overload
 def round(number: SupportsRound[_T], ndigits: SupportsIndex) -> _T: ...
-def setattr(__obj: object, __name: str, __value: Any) -> None: ...
+def setattr(__obj: object, __name: str, __value: object) -> None: ...
```

Comment:
While `setattr()` itself accepts any `__value`, conceptually it doesn't:

```python
class Foo:
    def __init__(self):
        self.x = 4
setattr(Foo(), "x", "abc")  # wrong type
```

Ideally, we'd be able to type is something like this:

```python
def setattr(__obj: type[_T], __name: AttributeOf[_T], __value: TypeOf[_T[__name]]) -> None: ...
```

We should use `Any` to mark that `__value`'s type can't really be expressed using our current type system.

Reply:
Sure, makes sense